bypros/storage/sqlstore/schema/schema.sql

```diff
@@ -0,0 +1,152 @@
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+CREATE SCHEMA cothority;
+ALTER SCHEMA cothority OWNER TO bypros;
+
+SET default_tablespace = '';
+SET default_table_access_method = heap;
+
+--
+-- Migration / Versioning
+--
+
+CREATE TABLE cothority.version (
+    version_id integer
+        PRIMARY KEY
+        GENERATED ALWAYS AS IDENTITY,
+
+    database_version integer NOT NULL,
```

Comment:
technically, schema_version 😅

Reply:
forgot: you might also want to include the very first row, saying ( version: 0, timestamp: now() )